scripts/zones/The_Shrine_of_RuAvitau/IDs.lua

```diff
@@ -35,6 +35,11 @@ zones[xi.zone.THE_SHRINE_OF_RUAVITAU] =
         ULLIKUMMI        = 17506418,
         OLLAS_OFFSET     = 17506667,
         KIRIN            = 17506670,
+        MOTHER_GLOBE     = 17506396,
+        SLAVE_GLOBES     =
+        {
+            17506397, 17506398, 17506399, 17506400, 17506401, 17506402
```

Comment:
You don't need to define more IDs here, when you can know for sure that:
```lua
local SLAVE_GLOBE_1 = ID.mobs.MOTHER_GLOBE + 1
local SLAVE_GLOBE_2 = ID.mobs.MOTHER_GLOBE + 2
local SLAVE_GLOBE_3 = ID.mobs.MOTHER_GLOBE + 3
local SLAVE_GLOBE_4 = ID.mobs.MOTHER_GLOBE + 4
local SLAVE_GLOBE_5 = ID.mobs.MOTHER_GLOBE + 5
local SLAVE_GLOBE_6 = ID.mobs.MOTHER_GLOBE + 6
```

So:
```lua
    for i = 1, 6 do
        local slaveGlobe = GetMobByID(ID.mob.MOTHER_GLOBE + i)
        -- ...
    end
```

Reply:
I'd rather not rely on convention where possible, but I'll make some change.

Reply:
Resolved?